app/models/concerns/deprecated_group_features.rb

```diff
@@ -0,0 +1,34 @@
+require 'active_support/concern'
+
+#### NB: These methods are deprecated and should not be relied upon going forward
+module DeprecatedGroupFeatures
```

Comment:
Yea, Overall, I'm into it.. but I feel like the naming needs a little more work. Can it start with group somehow? Maybe app/models/concerns/group_visible_to.rb

I think the deprecated thing is going to be a once off.. So thinking about where were working with features we want to keep.. I think a patten of model name then feature name will work well.